infrastructure/main.tf

```diff
@@ -126,12 +126,12 @@ module "claim-store-api" {
     CLAIM_STORE_DB_NAME = "${var.database-name}"
     CLAIM_STORE_DB_CONNECTION_OPTIONS = "?ssl"
 
-    SCHEDULER_DB_HOST = "${module.database.host_name}"
-    SCHEDULER_DB_PORT = "${module.database.postgresql_listen_port}"
-    SCHEDULER_DB_NAME = "${module.database.postgresql_database}"
-    SCHEDULER_DB_USERNAME = "${module.database.user_name}"
-    SCHEDULER_DB_PASSWORD = "${module.database.postgresql_password}"
-    SCHEDULER_DB_CONNECTION_OPTIONS = "?ssl"
+    CMC_DB_HOST = "${module.database.host_name}"
```

Comment:
If these are required to set for local environments, please update readme accordingly.